docs/UserGuide.md

```diff
@@ -273,23 +284,30 @@ Example:<br>
 
 ### Filtering tutors from the matched tutors: `filter`
 
-Filters tutor(s) from the list of matched tutors.
+Filters tutor(s) from the list of **matched** tutors.
 
-Format: <br>`filter t [n/NAME] [g/GENDER] [q/QUALIFICATION]`
+Format: <br>`filter [n/NAME] [g/GENDER] [q/QUALIFICATION]`
 - The search is case-insensitive. <br>
   e.g. `n/hans` will match `Hans`.
 - Only the last name given is searched. <br>
-  e.g. `filter t n/John n/Amy` will find names that match `Amy`.
+  e.g. `filter n/John n/Amy` will find names that match `Amy`.
 - Only full words will be matched. <br>
   e.g. `n/Han` will not match `Hans`, and `n/Han Solo` will not match `Han Dan`.
 - **At least one** of the optional fields must be provided.
 
+<div markdown="block" class="alert alert-warning">
+<b>:heavy_exclamation_mark: Caution:</b> <br> 
+
+* The use of `filter` should be preceded by a `match`. i.e. `filter` cannot be used on an empty Match List.
```

Comment:
i dont think this makes sense cause the user can do a `filter q/0` and then `filter g/F` without having to do a  `match` in between

Reply:
Okay, maybe I can just state that "`filter` cannot be used on an empty Match List."?

Reply:
sounds good!